scripts/v6/views/input-forms.sql

```diff
@@ -0,0 +1,24 @@
+-- Unfortunately SQL Server expects view definitions in seperate "statements", and it seems that
+-- any statements in the same file have to be separated by the SQL Server propietry GO keyword - which
+-- fails when executed over JDBC. So, our hand is forced, and we have to include views in separate 
+-- files here.
+
+CREATE View [dbo].[InputFormView] AS
+SELECT
+	AssetFinFormRef.Form_Nr,
+	AssetFinFormRef.Ext_Form_Reference,
+	AssetFinFormRef.Creator,
+	AssetFinFormRef.Form_Level,
+	AssetFinFormInput.*,
+	[Issue_DateTime], [Component_ID],	[Asset_Group_Name],	[Requested_Name], [Requested_Date],	[Authorised_Name],[Authorised_Date],
+	[Custodian_Name], [Custodian_Date],	[ExManager_Name],[ExManager_Date],[Captured_Name],[Captured_Date],[Information],[Instructions],
+	[Rec_Action],[Carrying_Value],[Value_in_Use],[Fair_Value_Less],[Recoverable_Amt],[Police_Report_Nr],[Take_on_Date],[Reason_Code],
+	[form_timestamp] [timestamp]
+FROM
+	AssetFinFormInput
+JOIN
+	AssetFinFormRef on AssetFinFormInput.Form_Reference = AssetFinFormRef.Form_Reference
+JOIN
+	AssetFinForm on AssetFinForm.Form_Nr = AssetFinFormRef.Form_Nr
+JOIN
+	AssetFinFormState on AssetFinFormState.Form_Level = AssetFinFormRef.Form_Level AND AssetFinFormState.Workflow = 'GENERIC';
```

Comment:
We can look at the tests using this view, and change them id required, though I don't think it's essential right now, as they aren't specifically testing the workflow state itself.

As for the rollup stored proc tests, technically the new values would determine how they determine whether or not to select SCOAJournal entries by CommittedToRegister=1 or CommittedToRegister=0, but the tests express this explicitly as params passed to the stored procs.